components/Create/index.tsx

```diff
@@ -105,7 +107,7 @@ export const Create = ({
               0,
               vest.toString(),
               epoch.toString(),
-              !!IS_PROD ? 'mainnet' : 'calibrationnet'
+              !!isProd ? 'mainnet' : 'calibrationnet'
```

Comment:
I think we still have the PR open to remove the `createMultisig` and I'm not sure if this argument matters. But shouldn't this be the network name from the Environment?

Reply:
These are hardcoded strings in the zondax library that (unfortunately) didn't line up with our naming (besides mainnet). This should operate more like "coinType" - its EITHER mainnet or calibrationnet, so i don't think we should use the NetworkName for this